tt1-blocks/block-templates/index.html

```diff
@@ -9,7 +9,7 @@
 		<!-- wp:query-loop -->
 		<!-- wp:post-title {"isLink":true} /-->
 
-		<!-- wp:post-content {"align":"full"} /-->
+		<!-- wp:post-content /-->
```

Comment:
With this in place, posts on the front-end extended beyond the edges of the screen. So I've removed it for now. Should probably re-evaluate once I can get the front-end alignments working right in general though.

Reply:
you should add `{ inherit: true }` to all post content in the templates if they are at the root of templates..